app-doc/heirloom-doctools/heirloom-doctools-160308.ebuild

```diff
@@ -0,0 +1,215 @@
+# Copyright 1999-2009 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: /var/cvsroot/gentoo-x86/app-doc/heirloom-doctools/Attic/heirloom-doctools-080407-r2.ebuild,v 1.3 2011/01/20 04:42:43 flameeyes dead $
+# revived from n-t-roff github fork.
+
+EAPI=6
+
+inherit flag-o-matic toolchain-funcs multilib
+
+DESCRIPTION="Classic Unix documentation tools ported from OpenSolaris"
+HOMEPAGE="http://heirloom.sourceforge.net/doctools.html"
+SRC_URI="https://github.com/n-t-roff/heirloom-doctools/releases/download/${PV}/${P}.tar.bz2"
+
+LICENSE="CDDL"
+
+SLOT="0"
+
+KEYWORDS="~amd64 ~x86"
+
+IUSE="cxx"
+
+RDEPEND="!sys-apps/groff"
+DEPEND="sys-devel/flex
+	virtual/yacc"
+
+src_prepare() {
+	# Make sure that C++ code is built with CXXFLAGS and not CFLAGS.
+	find . -name Makefile.mk -exec \
+		sed -i \
+			-e '/(CCC)/s:CFLAGS:CXXFLAGS:' \
+		{} +
```

Comment:
|| die.

Also, is this still necessary with the fork? Maybe submit a patch to the new upstream.